### ISSUE TITLE
Tutorial - more photos

### ISSUE THREAD

**Issue by ValeErre87** [geotagx/geotagx-theme#131]
I have received feedback from volunteers saying that sometime the photo used for the tutorial is not really representative for the purpose of the project or that they would like to be presented with more than one photo before starting to contribute to be confident about their ability to answer.

I think we should consider the idea of giving the chance to take the same tutorial with more than one photo (all photos showing different issues, as what the volunteers reported is a lack of sampling - e.g. ).

If we implement this, we need to consider if completing the additional tutorials should be mandatory or optional. In my opinion, it should be mandatory just for the first one and optional for the additional photos.

To sum up:
ACTION:
Provide more than one photo in the tutorial so that Analysts are presented with more sampling.
It could be mandatory to complete the first tutorial, and just optional to complete the others.
